### PR TITLE
[MULTIARCH-3707] OCP 4.14: Installation doc updates for IBM Power

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/MULTIARCH-3707

Draft: https://docs.google.com/document/d/11bqp8ogDLsCorOQnbLUopKdTmSuhkkbwMYhO1qRGOxs/edit

Link to docs preview: https://65854--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/minimum-ibm-power-system-requirements.adoc [hunk 1]
```diff
@@ -19,7 +19,7 @@ Support for {op-system} functionality for all IBM Power8 models, IBM Power AC922
 [discrete]
 == Hardware requirements
 
-* Six IBM Power bare metal servers or six LPARs across multiple PowerVM servers
+* Six LPARs across multiple PowerVM servers
```
Comment (lahinson) [inline modules/minimum-ibm-power-system-requirements.adoc:22]: As far as I can tell, this is the first instance of "LPARs" in this assembly. Will most users know what an LPAR is, or do we need to spell it out for the first instance?

File: modules/rhcos-enabling-multipath-day-1-power.adoc [hunk 2]
```diff
@@ -7,7 +7,7 @@
 [id="rhcos-enabling-multipathday-1-power_{context}"]
 = Enabling multipathing with kernel arguments on {op-system}
 
-In {product-title} 4.9 or later, during installation, you can enable multipathing for provisioned nodes.  {op-system} supports multipathing on the primary disk. Multipathing provides added benefits of stronger resilience to hardware failure to achieve higher host availability.
+In {product-title} version {product-version}, during installation, you can enable multipathing for provisioned nodes.  {op-system} supports multipathing on the primary disk. Multipathing provides added benefits of stronger resilience to hardware failure to achieve higher host availability.
```
Comment (lahinson) [inline modules/rhcos-enabling-multipath-day-1-power.adoc:10]:
```suggestion
In {product-title} version {product-version}, during installation, you can enable multipathing for provisioned nodes. {op-system} supports multipathing on the primary disk. Multipathing provides added benefits of stronger resilience to hardware failure to achieve higher host availability.
```

**Review by lahinson — COMMENTED** [review]
@alishaIBM I had one question about terminology, and I suggested one tiny edit, but otherwise, LGTM :)

File: modules/minimum-ibm-power-system-requirements.adoc (hunk 1 above)
Comment (alishaIBM) [reply modules/minimum-ibm-power-system-requirements.adoc]: @lahinson updated as:
```
* Six logical partitions (LPARs) across multiple PowerVM servers
```
fyi: @manojnkumar

**Review by sheriff-rh — APPROVED** [review]
LGTM! Merging.

**Pull request merged**